rocky/katalogus/templates/partials/plugin_tile.html

```diff
@@ -4,20 +4,24 @@
 <div role="group" id="plugin_{{ plugin.id|slugify }}">
     <img src="{% url "plugin_cover" organization_code=organization.code plugin_id=plugin.id %}"
          alt="boefje placeholder image" />
-    <div class="horizontal-view scan-intensity">
-        <span class="de-emphasized">{% translate "Scan level:" %}</span>
-        <ul class="level-indicator l{{ plugin.scan_level }}">
-            {% for i in "1234"|make_list %}<li></li>{% endfor %}
-        </ul>
-    </div>
-    <p>
-        <strong>{{ plugin.name }}</strong>
+    <p class="plugin-title">
+        <strong>{{ plugin.name }}</strong><span class="label-plugin-type {{ plugin.type }}">{{ plugin.type|title }}</span>
     </p>
-    <p>{{ plugin.description }}</p>
-    <span class="de-emphasized">{% translate "Publisher:" %} OpenKAT</span>
-    <a href="{% url "plugin_detail" organization_code=organization.code plugin_type=plugin.type plugin_id=plugin.id %}">{% translate "See details" %}</a>
+    {% if plugin.description %}<p class="plugin-description">{{ plugin.description }}</p>{% endif %}
+    {% if plugin.scan_level != None %}
+        <div class="horizontal-view scan-intensity">
+            <ul class="level-indicator l{{ plugin.scan_level }}">
+                {% for i in "1234"|make_list %}<li></li>{% endfor %}
+            </ul>
+        </div>
+    {% endif %}
+    <span class="plugin-author-link de-emphasized"><a target="_blank" href="https://openkat.nl">OpenKAT</a></span>
```

Comment:
Having this hardcoded doesn't really add much value? I would remove this until we have actually implemented this feature.